frontend/src/js/services/loaderService.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
@aayusharora : I think this is an auto generated comment. Can we please remove this ?